app/assets/javascripts/bastion/components/global-notification.service.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * @ngdoc service
+ * @name  Bastion.components.service:GlobalNotification
+ *
+ * @description
+ *  Service to display a global notifcation
+ */
+angular.module('Bastion.components').service("GlobalNotification", [
+    function () {
+        this.errorMessages = [];
+        this.successMessages = [];
+
+        this.setSuccessMessage = function (message) {
+            this.successMessages.push(message);
```

Comment:
Agreed it could be added outside of this PR and it should inevitably be consistent with the rest of the application (aka what do Foreman pages do?)